source/installation/index.rst

```diff
@@ -34,9 +34,39 @@ By default, it will perform a graphical installation, unless you use the msiexec
 Large Installations
 ^^^^^^^^^^^^^^^^^^^
 
+Consider a scenario where the GLPI Agent application needs to be installed on large and diverse range of Windows systems. A VBScript can be useful in this scenario to perform the following tasks:
+
+- Install it silently taking variables from the installation script previously configured.
+- Check for the presence of FusionInventory and OCS Inventory agents and uninstall them if you need it.
+- Configure application settings based on `commands that can be explored here <https://glpi-agent.readthedocs.io/en/latest/installation/windows-command-line.html#command-line-parameters>`.
```

Comment:
I think I missed to save that suggestion in my previous review. Actually the link should be the same than the one changed on l.52:
```suggestion
- Configure application settings based on `command line parameters that can be explored here <windows-command-line.html#command-line-parameters>`_.
```